README.md

```diff
@@ -47,4 +47,6 @@ Usage
 
 ```docker run -v /etc/sysconfig/:/etc/sysconfig/ MonsantoCo/etcd-aws-cluster```
 
+Or pass in REGION (default is 'us-east-1')
 
+```docker run -e REGION='us-west-2' -v /etc/sysconfig/:/etc/sysconfig/ MonsantoCo/etcd-aws-cluster```
```

Comment:
More precise, you should never add quotes like you would in a shell script in a systems unit file. The unit file is parsed by systems/fleet and not the shell.

The quotes end up as part of the variable value